cms/djangoapps/contentstore/tests/test_orphan.py

```diff
@@ -128,3 +131,50 @@ def test_not_permitted(self, default_store):
         self.assertEqual(response.status_code, 403)
         response = test_user_client.delete(orphan_url)
         self.assertEqual(response.status_code, 403)
+
+    def test_orphan_parents(self):
+        store = self.store
+        with store.default_store(ModuleStoreEnum.Type.split):
```

Comment:
If I'm understanding this correctly, you want a course tree that looks like:
```
 course
    |
 chapter
   |
 seq     seq
    \    /
     html
```

And you want to make sure that calling `get_parent` on `html` never returns the orphan parent? We actually have such an element already in `TestOrphanBase`: https://github.com/edx/edx-platform/blob/master/cms/djangoapps/contentstore/tests/test_orphan.py#L48-L52.

Just use that one